packages/core/admin/server/services/token.js

```diff
@@ -61,10 +61,20 @@ For security reasons, prefer storing the secret in an environment variable and r
   }
 };
 
+/**
+ * Create a random 6 digit verification code used for multi-factor authentication
+ * @returns {number}
+ */
+const createVerificationToken = () => {
+  return  Math.floor(100000 + Math.random() * 900000);
```

Comment:
mmmm, we probably need to do a bit of research here to see if this is a safe and proper way to generate a 2FA code

Reply:
I don't think we can move forward with this. We will need to find a secure solution for this. I asked some of the other engineers and we were thinking that you could possibly use the node `crypto` package. Wdyt?